templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -220,6 +220,24 @@
             },
             "Retry": ${retry_statement},
             "ResultPath": null,
+            "Next": "Insert into post-ingest table"
+          },
+          "Insert into post-ingest table": {
+            "Type": "Task",
+            "Resource": "arn:aws:states:::dynamodb:putItem",
+            "Parameters": {
+              "TableName": "${post_ingest_table_name}",
+              "Item": {
+                "ioRef": {
```

Comment:
Just curious, how come we are using `ioRef` instead of assetId?

Reply:
So the confirmer doesn't have to contact Preservica to look up the IO ref. The IO ref is available in the reconciler so we may as well send it.